components/select/select-multiple-value.tsx

```diff
@@ -1,26 +1,33 @@
 import React from 'react'
 import useTheme from '../styles/use-theme'
 import Grid from '../grid'
+import SelectClearIcon from './select-icon-clear'
 
 interface Props {
   disabled: boolean
+  onClear: () => void | null
```

Comment:
got an error: `Type error: Type '(() => void) | null' is not assignable to type '() => void | null'.`